C/CUDA/CUDA_Driver/init.jl

```diff
@@ -72,8 +72,15 @@ function try_driver(driver, deps)
 
         exit(0)
     """
-    success(`$(Base.julia_cmd()) --compile=min -t1 --startup-file=no -e $script $driver $deps`)
+    # make sure we don't include any system image flags here since this will cause an infinite loop of __init__()
+    success(`$(julia_cmd_without_sysimage()) --compile=min -t1 --startup-file=no -e $script $driver $deps`)
 end
+
+# removes the system image argument from the Julia command to force the session to use the default
+function julia_cmd_without_sysimage()
+    Cmd(filter(e -> !startswith(e, "-J") && !startswith(e, "--sysimage"), Base.julia_cmd().exec))
```

Comment:
`!startswith(r"-J|--sysimage")` should make this shorter. I'd just move it inline, a separate function doesn't seem too useful here, as we'll never call this multiple times.